src/Common/Editor/Full_Site/Assets.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace TEC\Common\Editor\Full_Site;
+
+use \Tribe__Main as Common;
+
+/**
+ * Class Assets
+ *
+ * @since   TBD
+ *
+ * @package TEC\Common\Editor\Full_Site
+ */
+class Assets extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * Binds and sets up implementations.
+	 *
+    * @since TBD
+	 */
+	public function register() {
+		$plugin = Common::instance();
```

Comment:
Is this required and not done as part of `Main`? If not required, then the whole service provider should be dropped.